From efa1bac65455e9858090aa161ab08ab6ce868063 Mon Sep 17 00:00:00 2001
From: Karl Wette <karl.wette@ligo.org>
Date: Wed, 17 Aug 2022 18:35:33 +1000
Subject: [PATCH 2/2] test_create_solar_system_ephemeris_python.sh: set j as a
 constant input variable

- This was probably using some gawk extension before, but fails on MacOS
---
 .../test_create_solar_system_ephemeris_python.sh              | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lalpulsar/bin/CreateEphemeris/test_create_solar_system_ephemeris_python.sh b/lalpulsar/bin/CreateEphemeris/test_create_solar_system_ephemeris_python.sh
index f6aa8f0a2b..db366412e4 100644
--- a/lalpulsar/bin/CreateEphemeris/test_create_solar_system_ephemeris_python.sh
+++ b/lalpulsar/bin/CreateEphemeris/test_create_solar_system_ephemeris_python.sh
@@ -26,8 +26,8 @@ for i in `seq 1 10`; do
         5|6|7)  rmsmax='1e-15';;        # velocity
         8|9|10) rmsmax='1e-20';;        # acceleration
     esac
-    paste earth98.txt earth98-python.txt | awk -v "i=$i" -v "rmsmax=$rmsmax" '
-        BEGINFILE { j = i + 10; rms = 0 }
+    paste earth98.txt earth98-python.txt | awk -v "i=$i" -v "j=$((i+10))" -v "rmsmax=$rmsmax" '
+        BEGINFILE { rms = 0 }
         { rms += ($i - $j)^2 }
         ENDFILE { rms = sqrt(rms / NR); printf "i=%i rms=%g rmsmax=%g\n", i, rms, rmsmax; exit ( rms <= rmsmax ? 0 : 1 ) }
     '
-- 
2.32.0 (Apple Git-132)

